### PR TITLE
[MIG] stock_by_warehouse_sale: Migration to 16.0 T#63809

### PR DESCRIPTION
**Pull request by rolandojduartem**
**Before recompute:**

Screenshot from 2023-03-17 16-31-58

**After recompute:**

Screenshot from 2023-03-17 16-32-08

### CODE REVIEW

**Comment by rolandojduartem** [comment]
Hi, @luisg123v could you review this migration? this PR goes after [!1602](https://github.com/Vauxoo/addons-vauxoo/pull/1602)

cc @desdelinux

**Comment by rolandojduartem** [comment]
@desdelinux, for the functional review, you could use the runbot on https://git.vauxoo.com/vauxoo/dhm/-/merge_requests/130

**Review by luisg123v — APPROVED** [review]
LGTM :+1: 

Just waiting for #1602 to merge this one.

File: stock_by_warehouse_sale/views/sale_order_views.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="name">sale.order.stock.warehouse</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_line']/form//field[@name='sequence']" position="after">
+                <div class="text-center alert alertdialog" groups="stock.group_stock_multi_warehouses">
```
Comment (luisg123v) [inline stock_by_warehouse_sale/views/sale_order_views.xml:10]: `role` not required here?

Comment (rolandojduartem) [reply stock_by_warehouse_sale/views/sale_order_views.xml:10]: In this case it is not required because it is an alertdialog and not an alert-info

**Review by desdelinux — APPROVED** [review]
LGTM

cc @luisg123v @hugho-ad

**Comment by rolandojduartem** [comment]
@luisg123v rebased!

**Pull request merged**